docs/General/on-the-server.rst

```diff
@@ -0,0 +1,109 @@
+will last. This program will wait for players to connect and handle their moves in the exact same
+way as in a single player game. In fact, Freeciv21 always uses a server, even when there is only
+one player!
+
+Installation
+============
+
+Installing Freeciv21 on a server is done the normal way, as documented in :doc:`/General/install`.
+Because the official packages come with the complete game, they will pull a number of dependencies
+that are normally not used on servers (for instance, a display server). This software is installed
+for packaging reasons, but will not be used.
+
+Starting the Server
+===================
+
+Once Freeciv21 is installed, it's immediately ready to run a simple server, which one can do by
+running the following command:
+
+.. code-block:: sh
+
+    freeciv21-server
+
+This starts a server listening on the port traditionally used for Freeciv21, 5556. You can provide
+a custom port number by passing it to the ``-p`` argument:
+
+.. code-block:: sh
+
+    freeciv21-server -p 5000
+
+A server started with this command can be reached by pointing the game to port 5000 of your domain
+name. We suggest to start the server from within a terminal multiplexer such as ``tmux`` or
+``screen``, which will let it run independently.
+
+User Permissions
+================
+
+Freeciv21 supports several access levels for players connected to a server, restricting which
+commands they are allowed to run:
+
+``none``
+    The user may not issue any command.
+
+``info``
+    The user may use informational commands only.
+
+``basic``
+    The user may use informational commands as well as commands affecting the game. Commands
+    affecting the game start a vote if more than one user is connected.
+
+``ctrl``
+    Same as ``basic``, but the vote is bypassed for commands affecting the game.
+
+``admin``
+    May use any command, except for ``quit``, ``rfcstyle``, and ``write``. This includes
+    potentially destructive commands such as ``save`` and ``fcdb`` --- use with care.
+
+``hack``
+    May use all commands without restriction.
+
+By default, users connected to your server have access level ``basic``. This can be changed using
+the ``cmdlevel`` command::
+
+    cmdlevel info new
+
+This command would grant access level ``info`` to any newly connecting player. A few more options
+are available; please refer to the documentation of ``cmdlevel`` for more information.
+
+.. note::
+    The ``take`` and ``observe`` commands require access level ``info`` only. Their use can be
+    restricted using the ``allowtake`` server option or, in more advanced setups, using the
+    ``user_take`` :doc:`fcdb </Coding/fcdb>` hook.
+
+Authentication
+==============
+
+By default, Freeciv21 servers are open to anyone without any sort of authentication. It is
+possible to make Freeciv21 ask for a password. There is also a built-in registration procedure
+with which one can create an account and set a password.
```

Comment:
If there's a built-in registration procedure, it should be either documented here, or linked to from here.

Reply:
It is not documented anywhere "yet". My personal expectation was to include in the server manual that has not been written yet. Not sure how we get around the ambiguity here.

Reply:
It's a function in fcdb, but the documentation we have for that is rather poor. It's intended to be linked in the last paragraph.

Reply:
Agreed. OK.